html-forms/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>HTML Forms</title>
+</head>
+
+<body>
+  <h1>Coding Form!</h1>
+  <form action="https://example.com/">
+    <p>Email
```

Comment:
this should be a label instead of a p tag. also, non of the `p` tags have closing `p` tags